doc/lib.rst

```diff
@@ -64,6 +64,11 @@ Core
 * `cpuinfo <cpuinfo.html>`_
   This module implements procs to determine the number of CPUs / cores.
 
+* `distros <distros.html>`_
+  This module implements the basics for OS distribution ("distro") detection and the OS's native package manager.
+  Its primary purpose is to produce output for Nimble packages, but it also contains the widely used **Distribution** enum
+  that is useful for writing platform specific code. It is imported implicitly by the compiler.
```

Comment:
It is not imported implicitly by the compiler! You can import it directly, it does not rely on compiler magic to work.

Reply:
My confusion is that the platform enum is defined by compiler magic, but the distros module is not.

I need to separate the two, but I am not sure where to put the documentation about the enum.